From 821b32e5c43aeb1b965d36a2fa4699c5c3074c73 Mon Sep 17 00:00:00 2001
From: Pavan S <psreenivas@habana.ai>
Date: Tue, 19 Dec 2023 08:40:35 +0200
Subject: [PATCH] [SW-166373] gaudi2: increase tpm init timeout

It is observed that on some host the prstn is
deasserted a bit later so increasing tpm
prstn timeout to 1sec

Signed-off-by: Pavan S <psreenivas@habana.ai>
Change-Id: I0fdc816b4ad150a966770457e2ea1d20314901b0
---

diff --git a/arch/gaudi2/common/tpm_auth.c b/arch/gaudi2/common/tpm_auth.c
index 27021c2..2ef53d0 100644
--- a/arch/gaudi2/common/tpm_auth.c
+++ b/arch/gaudi2/common/tpm_auth.c
@@ -41,7 +41,7 @@
 	return 0;
 }
 
-#define TPM_PRSTN_TIMEOUT_US	200000	/* 200msec */
+#define TPM_PRSTN_TIMEOUT_US	1000000	/* 1sec */
 int tpm_init(bool print)
 {
 	int ret;
